src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt

```diff
@@ -126,12 +137,92 @@ class PeerGradingService(
                     FROM Response resp
                     WHERE resp.interaction = :interaction
                 )
-        """.trimIndent()
+        """.trimIndent(), PeerGrading::class.java
         )
-            .setParameter("grader", user)
+            .setParameter("grader", grader)
             .setParameter("interaction", sequence.getResponseSubmissionInteraction())
             .resultList as List<PeerGrading>
 
+    /**
+     * Count the number of evaluations made by a user on a sequence.
+     *
+     * If the user has not made any evaluation, 0 is returned.
+     *
+     * @param grader the user who performed the evaluations.
+     * @param sequence the sequence.
+     * @return the number of evaluations.
+     */
+    fun countEvaluationsMadeByUser(grader: User, sequence: Sequence): Long {
```

Comment:
is this method still used somewhere in the code?

Reply:
With this commit [6040614](https://github.com/elaastic/elaastic-questions-server/pull/247/commits/6040614ff6ad49074ce6d5813de86bbfc56141c8), not anymore.

Should we remove it ?

Reply:
Fix in 42356046dc6b62e1b7649facb8f61542e5970bae